frontend/.vscode/settings.json

```diff
@@ -34,6 +34,9 @@
     "**/bower_components": true,
     "**/.cache-loader": true,
     "**/public/dist": true
+  },
```

Comment:
Perhaps leave a comment on the rule in question (since `.vscode/settings.json` supports them). 

```javascript
// legacy support
"eslint.autoFixOnSave": true,
```

Reply:
Done

Reply:
Great, makes it easy to remove later if we choose.